packages/caravan-psbt/src/psbtv0/utils.ts

```diff
@@ -143,7 +141,7 @@ export const convertLegacyOutput = (output: LegacyOutput): PsbtOutput => {
     value: new BigNumber(output.amountSats).toNumber(),
     bip32Derivation:
       output.bip32Derivation || getBip32Derivation(output.multisig),
-    witnessScript: output.witnessScript || output.multisig?.witness?.output,
+    witnessScript: output.witnessScript || output.multisig?.witness?.[0],
```

Comment:
Bit of a hack—`bitcoinjs-lib` payments api outputs an array of witness buffers. Just extracting the first element

Reply:
This one has me puzzled. Partly just because of all our legacy code and how this data gets passed around. Where is this object being created by bitcoinjs-lib in the first place? 

Second question is about the type. Was the type wrong before with a `witness.output`? And then for this buffer selection, is this saying that the witness could have multiple buffers? I want to make sure we don't accidentally end up just taking the first byte off of a single buffer. If this is actually coming from bitcoinjs-lib maybe we can type LegacyMultisig more explicitly using bitcoinjs-lib types to make this a little safer.